CONTRIBUTING.md

```diff
@@ -0,0 +1,160 @@
+# Contributing to BETER Feed Testing Sandbox (BFTS)
+
+## Introduction
+
+Thank you for considering contributing to the BETER Feed Testing Sandbox (BFTS) project! This document provides guidelines and instructions to help you contribute effectively.
+
+## Table of Contents
+
+1. [Code of Conduct](#code-of-conduct)
+2. [How to Contribute](#how-to-contribute)
+   - [Fork the Repository](#fork-the-repository)
+   - [Create a Branch](#create-a-branch)
+   - [Make Changes](#make-changes)
+   - [Test Your Changes](#test-your-changes)
+   - [Submit a Pull Request](#submit-a-pull-request)
+3. [Guidelines for Development](#guidelines-for-development)
+   - [Resource Requirements](#resource-requirements)
+   - [Load testing resources requirements](#load-testing-resources-requirements)
+   - [Versioning Process](#versioning-process)
+4. [Reporting Issues](#reporting-issues)
+5. [Getting Help](#getting-help)
+
+## Code of Conduct
+
+This project adheres to a [Code of Conduct](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to [email@example.com](mailto:email@example.com).
+
+## How to Contribute
+
+### Fork the Repository
+
+1. Fork the repository by clicking the "Fork" button at the top right corner of the repository page.
+2. Clone the forked repository to your local machine:
+
+   ```bash
+   git clone https://github.com/your-username/beter-feed-testing-sandbox.git
+   cd beter-feed-testing-sandbox
+   ```
+### Create a Branch
+
+1. Create a new branch for your feature or bug fix:
+
+```bash
+git checkout -b feature/your-feature-name
```

Comment:
For sure, this naming strategy contradicts github's suggested flow. Also, GiHub's flow is to clone the repo, to make changes and provide PR to the main repo.

Reply:
Rejected. Just 2 lines above this one the old naming.